include/mod_bosh.hrl

```diff
@@ -5,4 +5,5 @@
 
 -record(bosh_socket, {sid   :: mod_bosh:sid(),
                       pid   :: pid(),
-                      peer  :: {inet:ip_address(), inet:port_number()}}).
+                      peer  :: {inet:ip_address(), inet:port_number()},
+                      peercert :: undefined | binary()}).
```

Comment:
peercert -> peer_cert ??

Reply:
That's actually being consistent with `ssl:peercert`. I can change to `peer_cert` if you insist.

Reply:
Than please don't change it :)